### PR TITLE
fix: Dexterity and plone5

### PR DESCRIPTION
**Pull request by Gagaro**
Including travis configuration for Plone 4 and 5.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,11 +52,14 @@
...
           # -*- Extra requirements: -*-
+          'five.formlib',
       ],
       extras_require={
           'test': [
               'Products.CMFTestCase',
               'Products.PloneTestCase',
+              'plone.testing',
+              'plone.app.testing',
           ],
```
Comment (mauritsvanrees) [inline setup.py]: These new test dependencies are not actually used, right?

Comment (Gagaro) [inline setup.py]: They were needed because of https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/tests/PloneTestCase.py

Comment (mauritsvanrees) [inline setup.py]: I have pushed a change to import default_user from Products.PloneTestCase.

**Comment by mauritsvanrees** [comment]
Tests pass.  Works fine in an Archetypes Plone 4.3 project where I use it. Looks good to me. Thanks!

**Comment by mauritsvanrees** [comment]
I have released this as version 5.0.

File: setup.py (hunk 1 above)
Comment (Gagaro) [inline setup.py]: Wow I didn't pay attention that it was not from there.. My bad :disappointed: .

Thanks for the quick review/merge.

**Comment by mauritsvanrees** [comment]
I have released 5.1 which lists `plone.app.textfield` in its `install_requires`.

**Pull request merged**